exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -222,11 +225,16 @@ def log_status(body)
             klass_or_nil = ::Google::Protobuf::DescriptorPool.generated_pool.lookup(detail.type_name).msgclass
             detail.unpack(klass_or_nil) if klass_or_nil
           end.compact
-          OpenTelemetry.handle_error(message: "OTLP exporter received rpc.Status{message=#{status.message}, details=#{details}}")
+          OpenTelemetry.handle_error(message: "OTLP exporter received rpc.Status{message=#{status.message}, details=#{details}}, uri #{@uri_string}")
         rescue StandardError => e
           OpenTelemetry.handle_error(exception: e, message: 'unexpected error decoding rpc.Status in OTLP::Exporter#log_status')
         end
 
+        def log_request_failure(response_code, labels = {})
+          OpenTelemetry.handle_error(message: "OTLP exporter received http.code=#{response_code} for uri: '#{@uri_string}' in OTLP::Exporter#send_bytes")
```

Comment:
```suggestion
          OpenTelemetry.handle_error(message: "OTLP exporter received http.code=#{response_code} for uri='#{@uri}' in OTLP::Exporter#send_bytes")
```
use of `=` feels more consistent within the message.